manifests/plugin/openscap.pp

```diff
@@ -0,0 +1,66 @@
+# = Foreman Openscap plugin
+#
+# This class installs openscap plugin
+#
+# === Parameters:
+#
+# $configure_openscap_repo::  Enable custom yum repo with packages needed for foreman_openscap,
+#                             type:boolean
+#
+# $scap_client_module_dir::   if you set a directory here, installer will upload foreman_scap_client
+#                             puppet module into this directory for you
+#                             e.g. /etc/puppet/environments/production/modules
+#
+class foreman::plugin::openscap (
+  $configure_openscap_repo = false,
+  $scap_client_module_dir  = undef,
+) {
+  case $::osfamily {
+    'RedHat': {
+      validate_bool($configure_openscap_repo)
+
+      if $configure_openscap_repo {
+        case $::operatingsystem {
+          'fedora': {
+            $repo = 'fedora'
+          }
+          default: {
+            $repo = 'epel'
+          }
+        }
+
+        if $::osreleasemajor == undef {
```

Comment:
originally I thought I could [reuse this](https://github.com/theforeman/puppet-foreman/blob/master/manifests/install/repos/extra.pp#L9) but after lint changes, it does not make sense, I'll remove it